vita3k/modules/SceGxm/SceGxm.cpp

```diff
@@ -1612,7 +1612,12 @@ EXPORT(bool, sceGxmProgramIsDiscardUsed, const SceGxmProgram *program) {
 EXPORT(bool, sceGxmProgramIsEquivalent, const SceGxmProgram *programA, const SceGxmProgram *programB) {
     assert(programA);
     assert(programB);
-    return UNIMPLEMENTED();
+
+    if (!programA || !programB)
+        LOG_ERROR(SCE_GXM_ERROR_INVALID_POINTER);
```

Comment:
```suggestion
        return RET_ERROR(SCE_GXM_ERROR_INVALID_POINTER);
```small error here

Reply:
it return error on log + send false

Reply:
doesn't return false, only way that an int casted to bool is false is when the int is 0, so this would return true even if it is false

Reply:
No, error value is négative, so it return false, if value is > 0 get true if value is < 1 get false

Reply:
http://cpp.sh/8pwas
This should prove it, as i said, booleans are only false if int is 0, if it isn't 0 it is true

Reply:
Yes sorry, so using return false and after get erreur